.buildkite/scripts/steps/checks/verify_fips_enabled.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/scripts/common/util.sh
+
+.buildkite/scripts/download_build_artifacts.sh
+
+echo --- Verify FIPS enabled
+
+NODE_BINARY="$KIBANA_BUILD_LOCATION/node/bin/node"
```

Comment:
Sorry, I moved this recently.  
```suggestion
NODE_BINARY="$KIBANA_BUILD_LOCATION/node/glibc-217/bin/node"
```

Reply:
[8bd94f9](https://github.com/elastic/kibana/pull/183777/commits/8bd94f919b8c8b6e0e66020bf11d74f3d86bb510)